index.rst

```diff
@@ -73,9 +80,12 @@ endpoint site, each monitoring a different location.
 
 An Ingester is responsible for making ingest attempts to a data management
 system on the endpoint site.  Similarly to Finders, multiple Ingesters can be
-deployed on a given endpoint site to ingest images to different database
-management systems (e.g., Gen2 and  Gen3 Butler repositories).
-
+deployed on a given endpoint site to ingest images to different data management
+systems, e.g., Gen2 and Gen3 Butler repositories. In addition to LSST Butler
+repositories, an Ingester could also register files in place with `Rucio`_
+instead uploading them to it as the file in the storage area can possibly be
```

Comment:
instead of uploading the files to Rucio   (the "of" was the thing that caught my attention and then just might as well replace pronouns to avoid any confusion)